Framework/MDAlgorithms/src/TransformMD.cpp

```diff
@@ -160,8 +160,24 @@ void TransformMD::exec() {
       signal_t *signals = histo->getSignalArray();
       signal_t *errorsSq = histo->getErrorSquaredArray();
 
-      this->reverse(signals, histo->getNPoints());
-      this->reverse(errorsSq, histo->getNPoints());
+      size_t nPoints = histo->getNPoints();
+      size_t d4 = 1;
+      size_t nd = histo->getNumDims();
+      if (nd > 3) {
```

Comment:
These changes mean that I cannot scale my 4th or 5th dimension by -1, even if I want to

Reply:
This should be fixed now